Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
```

Comment:
Since the project supports 3.7+ do we want to default to the latest version here and use 3.8?